auditbeat/scripts/generate_config.go

```diff
@@ -40,7 +41,17 @@ func getConfig(file string) ([]byte, error) {
 		return nil, errors.Wrapf(err, "failed reading %v", file)
 	}
 
+	var arch string
+	switch *goarch {
+	case "i386":
+		arch = "32"
+	case "amd64":
+		arch = "64"
+	default:
+		return nil, fmt.Errorf("supporting only i386 and amd64 architecture")
+	}
 	data := map[string]interface{}{
+		"goarch":    arch,
```

Comment:
Perhaps this should be called `arch_bits` or something other than `goarch` since it's not exactly the GOARCH value?

Reply:
Make sense!